src/custom/components/WalletModal/index.tsx

```diff
@@ -12,12 +14,24 @@ const TermsWrapper = styled.div`
 `
 
 function CustomTerms() {
+  const history = useHistory()
+  const toggleWalletModal = useWalletModalToggle()
+
   return (
     <TermsWrapper>
       <Trans>
         By connecting a wallet, you agree to GnosisDAO&apos;s{' '}
-        <ExternalLink href={Routes.TERMS_CONDITIONS}>Terms &amp; Conditions</ExternalLink> and acknowledge that you have
-        read, understood, and agree to them.{' '}
+        <ExternalLink
+          href="#"
```

Comment:
nit: why not using just the regular URL here so it opens in new tab. It just feels easier, and user can read in the other tab and don't loose the context of the wallet.

Anyways, this also works for me

Reply:
```
<Link to={Routes.TERMS_CONDITIONS}>Terms &amp; Conditions</Link>
```
The Link from `'react-router-dom` looks more suitable here (with styled)